src/grub.cfg

```diff
@@ -57,4 +57,16 @@ if [ -f "/ignition.firstboot" ]; then
     set ignition_firstboot="ignition.firstboot ${ignition_network_kcmdline}"
 fi
 
+# Read ignition platform id from /ignition_platform_id
+set ignition_platform_id=""
```

Comment:
Accessing unset grub variables is ok and just returns empty string. I think this can be dropped.